resources/language/German/strings.po

```diff
@@ -59,11 +59,11 @@ msgstr "Bitter überprüfen Sie Ihre Kindersicherungs-PIN"
 
 msgctxt "#30008"
 msgid "Login failed"
-msgstr "Login nicht erfolgreich"
+msgstr "Anmeldung nicht erfolgreich"
 
 msgctxt "#30009"
 msgid "Please Check your credentials"
-msgstr "Bitte überprüfen Sie Ihre Login Daten"
+msgstr "Bitte überprüfen Sie Ihre Anmeldedaten"
```

Comment:
In Kodi, for the German translation, we never use direct speach (this is different from what we do in the English translation, but more common in Germany). It might be a good idea (for consistency reasons) to change this addons Germans translation to fit the German language style of "mother" Kodi.

Example:
Wrong: "Bitte überprüfen Sie Ihre Anmeldedaten"
Right: "Bitte die Anmeldedaten überprüfen"